### ISSUE TITLE
Is there a hotkey to temporarily disable NVDA?

### ISSUE THREAD

**Issue by rokit** [nvaccess/nvda#7882]
If not, I'd like to make this a feature request.

**Comment by Brian1Gaff** [comment]
Well that very much depends on what you mean by disable.

You can turn it off while running a program set a config so it stays off 
when that program runs or comes back on when switching apps or you can 
completely unload it and reload it with or without the sounds it makes.
 The temp key for desktops is  nvda/shift/s which is a toggle of course.
 Have a look att the hotkeys defined in the help section.

**Comment by rokit** [comment]
Toggle Speech Mode is what I was looking for, nvda + s. Sleep Mode may come
in handy as well. Thanks.

**Comment by PratikP1** [comment]
NVDA does not have a true sleep mode which would allow NVDA to be disabled in a particular application. This function is needed for developers testing NVDA and for completely speech-enabled applications. I would like to see NVAccess build this functionality so that app specific configuration profiles could use this setting to disable NVDA app-by-app basis.

**Comment by Brian1Gaff** [comment]
Well I use a very simple app module to do what you say but I'm sure you can 
use the setting to cause it to only be disabled in the app you are running 
anyway.
 Brian

**Comment by PratikP1** [comment]
The current setting only disables speech. What I propose is a sleep setting that completely disables NVDA in a given application—a setting that could be saved in configuration profile. Other than the ability to switch sleep mode on or off and launching configuration profiles, no other NVDA functionality would be present.

**Comment by Adriani90** [comment]
There is a sleep mode in NVDA, press nvda+shift+s in desktop keyboard layout or nvda+shift+y in laptop keyboard layout. Please create a feature according to the feature template where you describe exactly the behavior you would like to see. I am closing this one as works for me.
